kvmagent/kvmagent/test/shareblock_testsuite/test_sblk_metadata.py

```
@@ -0,0 +1,1006 @@
+        vol_lv = self._create_qcow2_lv('api_svol_%s' % tid)
+        try:
+            _create_qcow2_on_lv(base_lv)
+            _create_qcow2_on_lv(vol_lv, backing=base_lv)
+
+            sblk_vol = 'sharedblock:/%s/api_svol_%s' % (vgUuid, tid)
+
+            count = sblk_prefix_rebase_backing_files(
+                [sblk_vol],
+                'sharedblock:/%s' % vgUuid,
+                'sharedblock:/%s' % vgUuid,
+                translate_absolute_path_from_install_path, lvm)
```

Ruff (0.15.7)

[error] Undefined name `translate_absolute_path_from_install_path`

(F821)